packages/shared/ConsolePatchingDev.js

```diff
@@ -20,71 +20,76 @@ let prevError;
 let prevGroup;
 let prevGroupCollapsed;
 let prevGroupEnd;
+const enableDoubleLogging = global.__REACT_DEVTOOLS_ENABLE_DOUBLE_LOGGING__;
```

Comment:
What is `global` here? I don't think this would work in the browser. I think you'll want to do something like

```js
import {canUseDOM} from 'shared/ExecutionEnvironment';
```

and then if `canUseDOM` is true, read if from `window`.